dask/dataframe/csv.py

```diff
@@ -16,6 +18,12 @@
 
 delayed = delayed(pure=True)
 
+def GetUserMemory():
+    Memory = ps.virtual_memory().total
+    return Memory
+
+MEMORY_FRACTION = 10
+BLOCK_SIZE = GetUserMemory()/MEMORY_FRACTION
```

Comment:
Maybe instead of defining a function and then calling it, we can just call `BLOCK_SIZE = psutil.virtual_memory().total / MEMORY_FRACTION` here directly.